### PR TITLE
`optuna tell` with `--skip-if-finished`

### PR DESCRIPTION
**Pull request by hvy**
## Motivation

See description.

## Description of the changes

Fixes https://github.com/optuna/optuna/issues/3130.

### CODE REVIEW

File: optuna/cli.py [hunk 1]
```diff
@@ -855,16 +862,26 @@ def take_action(self, parsed_args: Namespace) -> int:
...
+        if parsed_args.skip_if_finished:
+            trials = study.get_trials(deepcopy=False)
+            trial = trials[trial_number]
+            assert trial.number == trial_number
+
+            if trial.state.is_finished():
+                self.logger.info(
+                    f"Skipped telling trial {trial_number} with values "
+                    f"{values} and state {state} since trial was already finished."
+                )
```
Comment (ytsmiling) [inline optuna/cli.py:879]: How about logging `trial.state` and `trial.values` as well as `state` and `values`?

**Review by ytsmiling — APPROVED** [review]
Thank you for creating this PR! This PR LGTM. I left a minor comment but I think this PR is already ready-to-merge even without addressing the comment.

File: optuna/cli.py (hunk 1 above)
Comment (hvy) [reply optuna/cli.py:879]: Sounds good. I'll fix that.

Comment (hvy) [reply optuna/cli.py]: Just fixed, PTAL if you have time over 🙇

Comment (ytsmiling) [reply optuna/cli.py]: Thank you for the swift fix!

**Review by ytsmiling — APPROVED** [review]
LGTM

**Review by HideakiImamura — APPROVED** [review]
Looks great to me.

**Pull request merged**